Resources/Prototypes/GameRules/midround.yml

```
@@ -15,6 +15,7 @@
       maxPicks: 10
     maxDifficulty: 2.5
   - type: AntagSelection
+    selectionTime: PrePlayerSpawn # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)